src/detail-page/DetailPage.tsx

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
 import { useParams } from 'react-router-dom';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 import { MainContainer } from '../layout/components/MainContainer';
 import { Tabs } from './components/tabs/Tabs';
 
 import { ITProfessions } from '../data/ITProfessions';
+import { ProfessionCard } from '../shared/components/ProfessionCard';
 
 export const TabContainer = styled('div')`
-    max-width: 960px;
-    width: 80%;
-    margin: 2em auto 0;
-    padding-bottom: 2em;
+  ${({ theme }) => css`
```

Comment:
Do we need the `css`? We can get `theme` from function props, like we did previously?